packages/data-service-generator/src/tests/__snapshots__/create-data-service-custom-path.spec.ts.snap

```diff
@@ -56,7 +56,7 @@ yarn-error.log*
     \\"build\\": \\"react-scripts build\\",
     \\"test\\": \\"react-scripts test\\",
     \\"eject\\": \\"react-scripts eject\\",
-    \\"docker:build\\": \\"docker build .\\"
+    \\"package:container\\": \\"docker build .\\"
```

Comment:
This change would probably need a change in the README files for the generated apps too.
@abrl91 do we mention docker:build in our docs too?

Reply:
Ah I think I was too rigoureus with the find and replace, we could leave it that way on the generated side, but it is your call!

Reply:
Let's do it for the generated app too since it's a good improvement. Just check the README for now

Reply:
I've checked the generated README and it is OK